src/csharp/Grpc.Core/ChannelCredentials.cs

```diff
@@ -104,40 +105,72 @@ internal override ChannelCredentialsSafeHandle CreateNativeCredentials()
         }
     }
 
+    /// <summary>
+    /// Callback invoked with the expected targetHost and the peer's certificate.
+    /// If a non-zero value is returned by this callback then it is treated as a
```

Comment:
"non-zero"  makes no sense here, it returns bool.

Reply:
Thanks, replaced by "false".